lib/elixir/lib/string_io.ex

```diff
@@ -39,10 +90,25 @@ defmodule StringIO do
       iex> StringIO.contents(pid)
       {"", ">"}
 
+      iex> {:ok, _contents} = StringIO.open("foo", fn(pid) ->
```

Comment:
We don't need to match on `{:ok, _contents}` here since we show the output of the function below.

Reply:
I removed all of them.